pootle/apps/pootle_store/models.py

```diff
@@ -1861,13 +1867,6 @@ def update(self, overwrite=False, store=None, only_newer=False, user=None,
             self.parse(store=store)
             return
 
-        disk_mtime = self.get_file_mtime()
-        if only_newer and disk_mtime == self.file_mtime:
```

Comment:
im not sure - i would think so - but its still used in Store.sync so i havent removed.

This clause defo does have to be removed as the "disk_mtime == self.file_mtime" is unreliable if the file is not on disk